README.markdown

```diff
@@ -100,6 +100,15 @@ Stick this in your vimrc:
 
 Note: Now start vim with plain `vim`, not `vim .`
 
+
+---
+> How can I open NERDTree as a tab window automatically when vim starts up on opening a directory?
```

Comment:
Sorry, I meant `b:NERDTree._type == "tab"` here. But it's really confusing for others.